### ISSUE TITLE
Release 1.15.1 Checklist

### ISSUE THREAD

**Issue by aphelionz** [hydroshare/hydroshare#2749]
- [x] Adding JSON-LD to HydroShare home page and resource detail pages #2685 (@aphelionz)
- [x] Hotfix for quota and google recaptcha #2696 (@hyi )
- [x] Add functionality to link specific web apps to the open with button for specific resources #2601 (@hyi )
- [x] fix the issue that quota from user zone is not accounted for #2750 (@hyi)
- [x] Resource type terms for HydroShare terms page #2676 (@sblack-usu)
- [x] extended metadata within File Metadata Tab doesn't work for editing or deleting #2733 (@sblack-usu)
- [x] HydroShare Python API returning Internal Server Error #2713 (@sblack-usu)

**Comment by aphelionz** [comment]
@ndebuhr for #1 yes, the foote rwas out of date. I have fixed.

For #2 @sblack-usu or @pkdash will be able to answer.

**Comment by hyi** [comment]
@ndebuhr For #2, refer to Model My Watershed web app tool resource on playground test VM here: https://playground.hydroshare.org/resource/2b548337c92a4ae5bb1b87b3fd35cf0f/ If you scroll down to extended metadata tab, you will see appkey-model-my-watershed key-value extended metadata set up for this web app tool resource. If you look at this resource https://playground.hydroshare.org/resource/d9290663a7f24b21a724b1e026542473/ you will see it has the same key-value extended metadata set up for it. As a result, that web app tool resource shows up in open with list of this resource due to the appkey extended metadata link. Hope this clarifies things and allows you to proceed with your testing.

**Comment by ndebuhr** [comment]
Thanks @hyi.  That's intuitive to me and looks good on the playground site.  However, I'm not getting the same behavior on beta.  For example, new Web App resources don't have the extended metadata, and I'm not seeing the extended metadata tab on existing Web App resources on the beta site.  The associated feature branch is merged into the 1.15.1 branch though.  @aphelionz Is this a database or perhaps configuration management difference/issue on the beta deployment?

**Comment by ndebuhr** [comment]
Thanks for the additional help on this @hyi.  I'm able to find that resource extended metadata on beta now.  So I have appkey=neal-test as a key value pair on both a test composite resource and a web app resource, just as you have in your playground site examples.  The addition of this extended metadata has been saved.  The ticket here https://github.com/hydroshare/hydroshare/issues/2601 and the google doc are a bit hard to follow, as far as rules for the association.  Should the private web apps and private composite resources be linkable (assuming they are owned by a single user, as in my test case)?  Otherwise, is some trigger needed to associate the web app and composite resources after they both have the matching appkey values?

**Comment by hyi** [comment]
@ndebuhr Yes, as long as the user has access to the web app tool resource and the resource meets the sharing status specified by the web app tool, the appkey matching will associate the resource with the web app tool. Do you need me to test this feature out on beta or you can take care of testing it?

**Comment by aphelionz** [comment]
You will need to check with @Lizabrazil about that. Ideally we would coordinate doc rollout with features

**Comment by hyi** [comment]
@ndebuhr I have tested appkey functionality on beta and everything works as expected. So I'll check it off from the verification list.
